.github/pull_request_template.md

```diff
@@ -86,7 +86,19 @@ Note that GitHub prefixes anchor names in markdown with "user-content-".
 
 #### Should this PR be included in the next patch release?
 
+`Yes` should be selected for bug fixes, documentation updates, and other small changes. `No` should be selected for new features and larger changes or if there is no recent minor/patch release. If you're unsure about the release classification of this PR, leave this unchecked to let the maintainers decide.
```

Comment:
> if there is no recent minor/patch release

I don't think users should have to reason about this. The MLflow maintainers (we) are in charge of figuring out when to release new significant features (minor releases) or bug fixes (patch releases). Can we remove this part?